x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/all/columns.tsx

```diff
@@ -228,9 +230,12 @@ export const getColumns = ({
         return value == null ? (
           getEmptyTagValue()
         ) : (
-          <LocalizedDateTooltip fieldName={i18n.COLUMN_LAST_UPDATE} date={new Date(value)}>
-            <FormattedDate value={value} fieldName={'last rule update date'} />
-          </LocalizedDateTooltip>
+          <FormattedRelativePreferenceDate
+            tooltipFieldName={i18n.COLUMN_LAST_UPDATE}
+            relativeThresholdInHrs={24}
```

Comment:
Could you please extract `24` into a constant, something like `DEFAULT_RELATIVE_THRESHOLD`, and reuse it instead of hardcoding the value?

Reply:
Great suggestion, thanks
I have extracted into `DEFAULT_RELATIVE_DATE_THRESHOLD`